### ISSUE TITLE
DGCA outputs rows filled with NA values

### ISSUE THREAD

**Issue by federicadannunzio1** [andymckenzie/DGCA#15]
I provided a matrix with 6 columns (3 samples for each condition) and 1340 rows to the ddcorAll() function in the DGCA package. The output correctly returns gene names, correlation values, and p-values for the first 5671 rows. However, after this point, the output consists of rows filled with NAs, continuing up to a total of 897130 rows. Several genes in the original matrix contain zero values in one of the two conditions, so I would expect some correlations and pvalues to be NA but I would expect those genes to be in the output at least in the "gene_name” column. Nonetheless I noticed that many gene names present in the original matrix are missing from the DGCA output. Could you please provide your assistance regarding this issue?

**Comment by andymckenzie** [comment]
Thank you for using DGCA and thank you for reporting this. I'm not sure what the problem is. To properly assess the issue, I'll need a minimal reproducible example. Could you please provide the following? 

1. A small subset of an input matrix that reproduces the problem (either from your data, or from a different/simulated data set that reproduces the same issue) 
2. Corresponding design matrix 
3. The exact code you're using to call ddcorAll()
4. The output of sessionInfo()

Basically if I can have a small data set that reproduces the issue, that will help me figure out what is going on. Thanks!
